src/renderer/hooks/useActiveTab.ts

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import { atom, useAtom } from "jotai";
+
+export const TabEnum = {
```

Comment:
Prefer using string literals instead of Enums in Typescript. for example: 
```ts
export type TabEnum = "Test Sequencer" | "Visual Python Script"
```